sdk/communication/azure-communication-callautomation/azure/communication/callautomation/aio/_call_connection_client_async.py

```diff
@@ -469,6 +473,7 @@ async def play_media_to_all(
         loop: bool = False,
         operation_context: Optional[str] = None,
         operation_callback_url: Optional[str] = None,
+        interrupt_call_media_operation: Optional[bool] = None,
```

Comment:
default to false

Reply:
As i mentioned, keeping the flag as bool or none to match the generated model.